lynx/common/actions/update_resources.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import Dict
+from dataclasses import field
+
+from lynx.common.actions.action import Action
+
+
+@dataclass
+class UpdateResources(Action):
```

Comment:
If some `Player` joins mid-game, they won't have any resources/points to put in their scoreboard until someone `Drops` resources in `DropArea`.

IMHO `UpdateResources` should be added to pending `Actions` upon new player joining. An alternative is adding resources/points to standard `GET` request answer, but that would be too heavy on the server.

Reply:
In general, in games, you should see what resources you can have (even if you don't have any)